pkg/DWSUtility.py

```diff
@@ -684,6 +684,11 @@ def do_delete_wfr(self, name):
             wfr_list.append(name)
 
         for wfr_name in wfr_list:
+            if self.config.wait:
+                wfr = self.dws.wfr_get(wfr_name)
```

Comment:
Seems slim, but is there any chance the get returns an exception here?

Reply:
the get will throw if not found 
https://github.com/NearNodeFlash/dwsutil/blob/d04f654e1d94e593fe503ee23db3e1faf9ad996b/pkg/Dws.py#L469

Reply:
Ah, I see that `run()` has this function wrapped in try/except.